packages/decoder/test/current/test/downgrade-test.js

```diff
@@ -411,11 +411,11 @@ function verifyAbiDecoding(decoding, address) {
   assert.lengthOf(decoding.arguments, 4);
   //we'll skip verifying the names as well
 
-  //first argument: {{x: 7, y: 5}, z: 3}
+  //first argument: {{x: 7, y: -5}, z: 3}
   assert.strictEqual(decoding.arguments[0].value.type.typeClass, "tuple");
   assert.deepEqual(
     Codec.Format.Utils.Inspect.unsafeNativize(decoding.arguments[0].value),
-    [[7, 5], 3]
+    [[7, -5], 3]
```

Comment:
what does the -5 mean?

Reply:
This is just a test fixture.  Since I changed that particular member from an unsigned type to a signed type, I figured I'd also change the test fixture to a negative number (as those are more likely to cause problems).